src/main/java/seedu/address/logic/Messages.java

```diff
@@ -12,6 +12,13 @@
  */
 public class Messages {
 
+    public static final String MESSAGE_NOT_FOUND_INDEX = "Error: Invalid Index";
+
+    public static final String MESSAGE_MISSING_INDEX = "Error: Missing Index";
+
+    public static final String MESSAGE_IMPOSSIBLE_INDEX = "Error: The parameter is not of the type positive integer";
```

Comment:
Seems like this can be used for features that uses indexes as well. Thank you for the initiative.